### ISSUE TITLE
add spenttime to issue

### ISSUE THREAD

**Issue by eprogres** [Netflix-Skunkworks/go-jira#47]
Hi,
is there any option to assign time to worklog in issue using this tool?
I try to play with edit command, but without success  

thanks
Peter

**Comment by coryb** [comment]
Sort of, but it is not in the default templates.  We can add it, but I am not sure what the exact format of the spenttime field is.  

You can export and edit the "edit" template with:
```
jira export-template -t edit
```
then the template file will be in `$HOME/.jira.d/templates/edit`

The to add a new field for spenttime you would add something like:
```
{{if .meta.fields.spenttime}}
spenttime: {{ .overrides.spenttime }}
{{end}}
```

Then the next time you run `jira edit ...` you will use the new template with the spenttime field.

There is more information on templates here:
https://github.com/Netflix-Skunkworks/go-jira/blob/master/README.md#templates

Test it out and let me know what changes you had to make to the templates to get them working and I will merge them in to the default templates.

-Cory

**Comment by coryb** [comment]
Actually I think this issue is a dup of #18, so I hope we will be able to get something for this soon.

**Comment by coryb** [comment]
This is done now and released in 0.1.6.

Usage to see current worklog:
```
jira worklog $ISSUE
```

Usage to add a worklog entry:
```
jira add worklog $ISSUE
```

The `add worklog` command will fire up an editor and allow you to enter a comment and timeSpent.  There is also a "started" field if you want to back-date the timeSpent, but you can leave it blank.  

The "timeSpent" field is in the format of "1h 15m".  

You can run `add worklog` without the editor via a command like this:
```
jira add worklog $ISSUE -m "work is hard" -T "1h 15m" --noedit
```

-Cory
